### ISSUE TITLE
Add option to remove seperator between events in widget

### ISSUE THREAD

**Issue by anmith** [SimpleMobileTools/Simple-Calendar#1730]
Hi,

I suggest an option to remove the thin line between events in the widget.

Especially on dark wallpapers and transparent widget background this fine grey line is (at least IMHO) annoying.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by anmith** [comment]
Hi tibbi,
it's the latest F-Droid version of Simple Calendar Pro 6.18.1, running on GrapheneOS.

**Comment by chrBrd** [comment]
Agreed - I have a solid black background and the lines make the widget look pretty janky.
